Gemfile

```diff
@@ -93,7 +93,7 @@ gem 'bootsnap', require: false
 gem 'geocoder'
 gem 'gmaps4rails'
 gem 'mimemagic', '> 0.3.5'
-gem 'paper_trail', '~> 12.1'
+gem 'paper_trail', '~> 15.1'
```

Comment:
This should also be done in a separate pull request. I don't know the changelog but there's probably a reason why we pinned the version.